pkg/controller/util.go

```diff
@@ -384,7 +384,8 @@ func MakeImporterPodSpec(image, verbose, pullPolicy string, podEnvVar *importPod
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: podName,
```

Comment:
If no longer using GenerateName, then `findImporterPod` should be refactored to simply do a get rather than list

Reply:
Good point done.